src/transformers/models/llama/modeling_llama.py

```diff
@@ -100,6 +100,21 @@ def __init__(self, dim, max_position_embeddings=2048, base=10000, device=None):
         inv_freq = 1.0 / (self.base ** (torch.arange(0, self.dim, 2, dtype=torch.int64).float().to(device) / self.dim))
         self.register_buffer("inv_freq", inv_freq, persistent=False)
 
+        # TODO: Remove in 4.40.
```

Comment:
why 4.40 here? This kind of version dependant removal would be for deprecation of a feature, but AFAICT in the PR comment we don't have an implemented fix which replaces this

Reply:
huh - OK. Won't this means things still break though?

Reply:
I don't think we can remove them, no 💔